apprentice_teacher.rb

```diff
@@ -1,39 +1,43 @@
-class ApprenticeTeacher
+require_relative 'person'
+require_relative 'teacher_module'
+
+class ApprenticeTeacher < Person
   attr_reader :age, :salary, :phase, :target_raise
   attr_accessor :name
 
+  include Teacher_stuff
+
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
+    # @age = options.fetch(:age, 0)
+    # @name = options.fetch(:name, "")
     @target_raise = 800
     @phase = 3
   end
 
-  def offer_high_five
-    "High five!"
-  end
+  # def offer_high_five
+  #   "High five!"
+  # end
 
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
+  # def set_phase(num)
+  #   @phase = num
+  #   "Cool, I've always wanted to teach phase #{num}!"
+  # end
 
   def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works. "
-    response += "*drops crazy knowledge bomb* "
-    response += "... You're welcome."
+    response = super
+    response.gsub!('*drops flat-out insane', '. *drops crazy')
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
+  # def salary=(new_salary)
+  #   puts "This better be good!"
+  #   @salary = new_salary
+  # end
 
-  def receive_raise(raise)
-    @salary += raise
-  end
+  # def receive_raise(raise)
+  #   @salary += raise
+  # end
 
   def set_performance_rating(rating)
```

Comment:
I'm curious why you refactored the `teach_stuff` method but not this one, since this one was the same except for a single variable (80 vs 90) and `teach_stuff` had more differences.